R/elasticsearch_parsers.R

```diff
@@ -628,6 +636,26 @@ es_search <- function(es_host
         log_fatal(msg)
     }
     
+    # Other input checks we don't have explicit error messages for
+    .assert(
+        assertthat::is.string(es_host)
+        , es_host != ""
+        , assertthat::is.string(es_index)
+        , es_index != ""
+        , assertthat::is.string(query)
+        , query != ""
+        , assertthat::is.string(scroll)
+        , scroll != ""
+        , assertthat::is.number(max_hits)
+        , max_hits >= 0
+        , assertthat::is.number(n_cores)
+        , n_cores >= 1
+        , assertthat::is.flag(break_on_duplicates)
+        , assertthat::is.flag(ignore_scroll_restriction)
```

Comment:
May want to check that these args aren't NA. Unfortunately `assertthat::is.flag(NA)` is TRUE because NAs are logical by default.

Reply:
ah yep, good catch